tiledb/sys-defs/src/lib.rs

```diff
@@ -193,3 +193,8 @@ pub type tiledb_vfs_mode_t = ::std::os::raw::c_uint;
 pub const tiledb_walk_order_t_TILEDB_PREORDER: tiledb_walk_order_t = 0;
 pub const tiledb_walk_order_t_TILEDB_POSTORDER: tiledb_walk_order_t = 1;
 pub type tiledb_walk_order_t = ::std::os::raw::c_uint;
+
+pub const tiledb_current_domain_type_t_TILEDB_NDRECTANGLE:
+    tiledb_current_domain_type_t = 0;
+#[doc = " The current domain type"]
```

Comment:
Was there a reason to go the attribute route instead of `///`? I don't care which is used, I'm just curious if I'm missing something.

Reply:
Just a bad copy/paste